set.js

```diff
@@ -0,0 +1,90 @@
+class Set {
+  constructor(array) {
+    this.values = array || [];
+  }
+  add(value) {
+    this.values.push(value);
```

Comment:
how does your set behave different than an array? Sets should not have duplicates?